packages/portal-proto/src/features/facets/NumericRangeFacet.tsx

```diff
@@ -782,12 +788,21 @@ const NumericRangePanel: React.FC<NumericFacetData> = ({
 }: NumericFacetData) => {
   const adjMinimum = minimum != undefined ? minimum : 0;
   const adjMaximum = maximum != undefined ? maximum : 999999;
+
+  const filter = hooks.useGetFacetFilters(field);
+  const [filterValues] = useMemo(() => {
```

Comment:
might wanna use `useDeepcompareMemo` given filter is an object or does it not matter here?

Reply:
Using useDeepcompareMemo might be better. The selector itself is not memorized so this potentially would mitigate it.